<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="TabBase64">
       <attribute name="title">
        <string>Base64</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>471</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="Tab_Text">
         <attribute name="title">
          <string>文本输入</string>
         </attribute>
         <widget class="QTextEdit" name="OutputBox_Base64">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>30</y>
            <width>341</width>
            <height>331</height>
           </rect>
          </property>
         </widget>
         <widget class="QTextEdit" name="InputBox_Base64">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>341</width>
            <height>331</height>
           </rect>
          </property>
         </widget>
         <widget class="QComboBox" name="ChoiceMode_Base64">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-10</y>
            <width>151</width>
            <height>51</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>Base64编码</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Base64解码</string>
           </property>
          </item>
         </widget>
         <widget class="QPushButton" name="ActionButton_Base64">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>370</y>
            <width>131</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>运行</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>180</y>
            <width>58</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>——&gt;</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="Tab_File">
         <attribute name="title">
          <string>文件输入</string>
         </attribute>
         <widget class="QLineEdit" name="Line_File_Input_Base64">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>721</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QTextEdit" name="Show_File_Input_Base64">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>70</y>
            <width>331</width>
            <height>251</height>
           </rect>
          </property>
         </widget>
         <widget class="QTextEdit" name="Show_File_output_Base64">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>70</y>
            <width>331</width>
            <height>251</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>180</y>
            <width>58</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>——&gt;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="Line_File_Output_Base64">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>340</y>
            <width>721</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="Button_File_Input">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>380</y>
            <width>131</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 64, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
          </property>
          <property name="text">
           <string>导入</string>
          </property>
         </widget>
         <widget class="QPushButton" name="Button_File_Action_Base64">
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>380</y>
            <width>131</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>运行</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="TabHaxi">
       <attribute name="title">
        <string>HaXi</string>
       </attribute>
       <widget class="QTextEdit" name="InputBox_Haxi">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>341</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="OutputBox_Haxi">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>40</y>
          <width>341</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>180</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>——&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ActionButton_Haxi">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>370</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>运行</string>
        </property>
       </widget>
       <widget class="QComboBox" name="ChoiceMode_Haxi">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>151</width>
          <height>51</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>SHA256</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MD5(过时的)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FNV1A64</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="ChoiceSalt_Haxi">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>0</y>
          <width>151</width>
          <height>51</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="TabNcm">
       <attribute name="title">
        <string>NCM</string>
       </attribute>
       <widget class="QLineEdit" name="LimeView_ncm">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>731</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QListWidget" name="ListShow_ncm">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>731</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="InputButton_ncm">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>390</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>导入</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ActionButton_ncm">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>390</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>运行</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="TabLan">
       <attribute name="title">
        <string>语言</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
